labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -64,9 +68,12 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean = 
+    seq match {
+      case Nil => true
+      case a::Nil => true
+      case a::b::xs => if (a >= b) false else checkValuesIncrease(b::xs)
```

Comment:
I hadn't even thought to go back and use pattern matching after getting to that lab, but this is a really nice way of doing this problem.